### PR TITLE
Add Alpine email client

### PR DESCRIPTION
**Pull request by daemonspudguy**
Fixes #

(let GitHub automatically close an issue when this pull request gets merged)

## Before you submit a pull request
This template is not neccessary when you do simple things like updating packages to the latest version. But in doubt, it's always better so provide some information.

## Description
Adding a new package

Works properly:
- [x] x86_64
- [ ] aarch64 (reasons why it doesn't)
Can't test aarch64
---

### CODE REVIEW

File: packages/alpine.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Alpine < Package                 # The first character of the class name must be upper case
+  description 'An easy-to-use email client.'
+  homepage 'http://alpine.x10host.com/alpine/'
```
Comment (cstrouse) [inline packages/alpine.rb]: If this is available via https, please update to use the secure version.

File: packages/alpine.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  version '2.22'
+  source_url 'http://alpine.x10host.com/alpine/release/src/alpine-2.22.tar.xz'
+  source_sha256 '849567c1b6f71fde3aaa1c97cf0577b12a525d9e22c0ea47797c4bf1cd2bbfdb'   # Use the command "sha256sum"
+
+  def self.build                   # the steps required to build the package
+    system "./configure", "--with-readline", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}", "--with-ssl-dir=/etc/ssl", "--with-ssl-include-dir=/usr/local/include", "--with-ssl-lib-dir=/usr/local/lib64", "--disable-nls"
```
Comment (cstrouse) [inline packages/alpine.rb]: We have a set of constants that are used to refer to commonly used paths such as `/usr/local/` (for prefix), `/usr/local/lib` or `/usr/local/lib64` (for libraries), etc. Please look at other packages for how we used them and look at the output of the `crew const` command.

An example would be: `--prefix=/usr/local` should be `--prefix=#{CREW_PREFIX}`

You may also want to look at the PKGBUILD from Arch Linux for this to get an idea of any other configure options that might make sense such as an initial configuration file.

File: packages/alpine.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Alpine < Package                 # The first character of the class name must be upper case
+  description 'An easy-to-use email client.'
```
Comment (cstrouse) [inline packages/alpine.rb]: Consider using a more descriptive description. We often just use the description from NetBSD's pkgsrc or Arch Linux's PKGBUILDs. Arch describes this package as: "The continuation of the Alpine email client from University of Washington"

**Comment by cstrouse** [comment]
@daemonspudguy Thanks for the PR! Just a few suggestions in the review that would be great if you could fix and push to this branch.

I'll test this on ARM and hopefully it will build.

**Comment by daemonspudguy** [comment]
@cstrouse What fixes?

**Comment by cstrouse** [comment]
@daemonspudguy The ones in the code review that I started on this PR. They should show up as yellow/beige boxes with comments tied to specific parts of the code you submitted.

**Comment by daemonspudguy** [comment]
There is no one in the Reviewers section.

File: packages/alpine.rb (hunk 1 above)
Comment (daemonspudguy) [reply packages/alpine.rb]: No HTTPS available

File: packages/alpine.rb (hunk 2 above)
Comment (cstrouse) [inline packages/alpine.rb]: `--with-readline` is not a valid configure option

**Comment by cstrouse** [comment]
This builds for me on aarch64.

**Comment by daemonspudguy** [comment]
Check now...

File: packages/alpine.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+class Alpine < Package                 # The first character of the class name must be upper case
+  description 'The continuation of the Alpine email client from University of Washington.'
+  homepage 'http://alpine.x10host.com/alpine/'
+  version '2.22'
+  source_url 'http://alpine.x10host.com/alpine/release/src/alpine-2.22.tar.xz'
+  source_sha256 '849567c1b6f71fde3aaa1c97cf0577b12a525d9e22c0ea47797c4bf1cd2bbfdb'   # Use the command "sha256sum"
+
+  def self.build                   # the steps required to build the package
+    system "./configure", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}", "--with-ssl-dir=/etc/ssl", "--with-ssl-include-dir=/usr/local/include", "--with-ssl-lib-dir=#{CREW_LIB_PREFIX}", "--disable-nls", "--with-system-pinerc=#{CREW_PROFILE}/etc/alpine.d/pine.conf", "--with-sysytem-fixed-pinerc=#{CREW_PREFIX}/etc/alpine/pine.conf.fixed"
```
Comment (uberhacker) [inline packages/alpine.rb]: Please put each configure option ending with a comma on a separate line.  One long line makes it hard to read without scrolling horizontally.  Also, please change `/usr/local` and `#{CREW_PROFILE}` to `#{CREW_PREFIX}`.

Comment (uberhacker) [reply packages/alpine.rb]: Also, change:
```
--with-sysytem-fixed-pinerc
```
to
```
--with-system-fixed-pinerc
```

File: packages/alpine.rb (hunk 2 above)
Comment (daemonspudguy) [reply packages/alpine.rb]: Done

File: packages/alpine.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  version '2.22'
+  source_url 'http://alpine.x10host.com/alpine/release/src/alpine-2.22.tar.xz'
+  source_sha256 '849567c1b6f71fde3aaa1c97cf0577b12a525d9e22c0ea47797c4bf1cd2bbfdb'   # Use the command "sha256sum"
+
+  def self.build                   # the steps required to build the package
+    system "./configure",
+    "--prefix=#{CREW_PREFIX}",
+    "--libdir=#{CREW_LIB_PREFIX}",
+    "--with-ssl-dir=/etc/ssl",
+    "--with-ssl-include-dir=/usr/local/include",
```
Comment (uberhacker) [inline packages/alpine.rb]: Please change:
```
--with-ssl-include-dir=/usr/local/include
```
to
```
--with-ssl-include-dir=#{CREW_PREFIX}/include
```

**Pull request merged**